canonical-websites/cn.ubuntu.com
# Fix IS8 navigation issues

## Done
Added the png fallback logo. Hidden the global navigation on older browsers. 

## QA
- Pull down this branch
- Run `make clean-all run`
- Go to http://localhost:8010/
- Check everything looks the same
- Go to browserstack and cn.ubuntu.com
- Check that the logo is not there and the global navigation pushed to page down
- Connect locally to browserstack
- Check that the logo  works and the global navigation is not visible

## Issue / Card
Fixes https://github.com/canonical-websites/cn.ubuntu.com/issues/63

## Screenshots
![10511892](https://cloud.githubusercontent.com/assets/1413534/23582758/5653981e-0129-11e7-9cd3-95aaaeae79b0.png)
